runtime/development/src/weights/pallet_anchors_v2.rs

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
Reminder to add actual weights.

Reply:
They are computed automatically before each RU, so we're safe even if we forget